controllers/certificate_lifecycle_controller.go

```diff
@@ -35,28 +35,31 @@ type CertificateLifecycle struct {
 }
 
 func (s *CertificateLifecycle) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
-	logger := log.FromContext(ctx)
+	log := log.FromContext(ctx)
 
-	logger.Info("starting CertificateLifecycle handling")
+	log.Info("starting CertificateLifecycle handling")
 
 	secret := corev1.Secret{}
-	if err := s.client.Get(ctx, request.NamespacedName, &secret); err != nil {
-		if k8serrors.IsNotFound(err) {
-			logger.Info("resource may have been deleted, skipping")
+	err := s.client.Get(ctx, request.NamespacedName, &secret)
+	if k8serrors.IsNotFound(err) {
+		log.Info("resource have been deleted, skipping")
 
-			return reconcile.Result{}, nil
-		}
+		return reconcile.Result{}, nil
+	}
+	if err != nil {
```

Comment:
We didn't check the error in the current `HEAD`, this is not a `chore` commit but rather a `fix` one since it's a bug.

Reply:
Thanks, I renamed the commit.